infra/docker/depot/internal/Dockerfile.prod

```diff
@@ -0,0 +1,122 @@
+FROM golang:latest
```

Comment:
This might just be an optimization that is overkill at the moment, but I was thinking. Does it make sense to create our own `go` docker image that could be like `gophr:golang` or something. It would basically be the templating that we use for all our of our go images (in herits from `golang:latest` includes all of the libgit2 installations for us etc..). It could clean up our images a ton, and make them way more readable and maintainable. Just a thought.